### ISSUE TITLE
Managing real world watchOS devices

### ISSUE THREAD

**Issue by mozeryansky** [facebook/idb#599]
I haven't been able to figure out how to use idb for my watchOS device automation. Is this supported or possible?

**Comment by c-ryan747** [comment]
I don't think this is something we've ever tried. I imagine it'll be possible to get it working, so if you want to take a crack at it we're happy to review some PRs!

cc @lawrencelomax
